NEWS.rst

```diff
@@ -2,6 +2,28 @@
 Release Notes
 =============
 
+----------------------------
+pkgcore 0.12.24 (2023-12-22)
+----------------------------
+
+- operations.format.fetch_base: try to use proxy values from domain as first
```

Comment:
Explain what this means? (How is it user-visible)?

Reply:
Hmm, you are correct - this was too cryptic.
How about now?